docs/pages/desktop-access/rbac.mdx

```diff
@@ -57,14 +57,15 @@ use wildcards (`"*"`) to match all desktop labels.
 
 Windows desktops acquire labels in two ways:
 
-1. The `host_labels` rules defined in the `windows_desktop_service` section of
-   your Teleport configuration file.
-2. Automatic `teleport.dev/` labels applied by Teleport (for desktops discovered
-   via LDAP only)
+1. Via the `host_labels` rules defined in the `windows_desktop_service` section
+   of your Teleport configuration file.
+2. Via LDAP (for desktops discovered via LDAP only)
 
-For example, the following `host_labels` configuration would apply the
-`environment: dev` label to a Windows desktop named `test.dev.example.com`
-and the `environment: prod` label to `desktop.prod.example.com`:
+### With `host_labels`
```

Comment:
```suggestion
### Via `host_labels`
```
Nit: Suggested to match the list that precedes the section and make it more explicit that this is expanding that list. (Alternatives would be to change "Via" to "With" in the list or to use "Using" in the list and headings.)